### PR TITLE
Lite route redesign

### PR DESCRIPTION
**Pull request by joelchoo**

### CODE REVIEW

**Comment by joelchoo** [comment]
https://projects.invisionapp.com/share/RGEY2TB9J#/screens/284336716

**Comment by LoneRifle** [comment]
=( is there no way we can derive lite stops from regular ones?

**Comment by joelchoo** [comment]
@LoneRifle what do you mean?

File: static/templates/tab-lite-detail.html [hunk 1]
```diff
@@ -3,81 +3,74 @@
...
-                 companyId: book.route.transportCompanyId,
-                 label: book.route.label})">
-            More Information
-            <i class="icon ion-ios-arrow-right"></i>
-          </div>
+      <div class="lite-stops">
+        <div ng-repeat="stop in book.route.stops"
+             class="lite-stop-container"
+             ng-click="setSelected($index)">
+          <div class="selector" ng-class="{ selected: $index === current || (!current && $index === 0)}"></div>
```
Comment (LoneRifle) [inline static/templates/tab-lite-detail.html]: `$index === (current || 0)`?

File: static/templates/tab-lite-detail.html [hunk 2]
```diff
@@ -3,81 +3,74 @@
...
+
     <form class="stops-form" name="stopsForm">
       <div class="route-info">
-        <div class="item item-icon-left item-text-wrap" ng-if="!book.hasTrips">
+        <div class="item item-icon-left item-text-wrap route-status" ng-if="!book.hasTrips">
           <span class="icon ion-android-alert"></span>
           There is no bus service scheduled for today.
         </div>
-        <div class="item item-icon-left item-text-wrap" ng-if="book.hasTrips && book.inServiceWindow===false">
+        <div class="item item-icon-left item-text-wrap route-status" ng-if="book.hasTrips && book.inServiceWindow===false">
```
Comment (LoneRifle) [inline static/templates/tab-lite-detail.html]: `!book.inServiceWindow` ?

**Pull request merged**